CruCentralCoast/Community/CommunityGroups/CommunityGroupDetailsVC.swift

```diff
@@ -7,41 +7,87 @@
 //
 
 import UIKit
+import MessageUI
 
-class CommunityGroupDetailsVC: UIViewController {
+class CommunityGroupDetailsVC: UIViewController, MFMessageComposeViewControllerDelegate {
+
+    
 
     @IBOutlet weak var bannerImageView: UIImageView!
-    @IBOutlet weak var genderLabel: UILabel!
-    @IBOutlet weak var dayLabel: UILabel!
-    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet weak var dayTimeLabel: UILabel!
     @IBOutlet weak var nameLabel: UILabel!
-    @IBOutlet weak var yearLabel: UILabel!
+    @IBOutlet weak var yearGenderLabel: UILabel!
     @IBOutlet weak var movementLabel: UILabel!
-    @IBOutlet weak var leaderNamesLabel: UILabel!
-    @IBOutlet weak var summaryLabel: UILabel!
+    @IBOutlet weak var joinCommunityGroupButton: CruButton!
     @IBOutlet weak var imageViewAspectRatioConstraint: NSLayoutConstraint!
-
+    
+    var leaderPhoneNumbers : [String] = []
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         self.navigationItem.largeTitleDisplayMode = .never
+        
     }
     
-    @IBAction func joinCommunityGroup() {
-        // TODO
-        self.presentAlert(title: "Join Community Group", message: "Coming Soon...")
+    @IBAction func didTapContactLeader() {
+        if MFMessageComposeViewController.canSendText() {
+            if self.leaderPhoneNumbers.isEmpty {
+                self.presentAlert(title: "Can't Contact Leader", message: "Sorry, there is no phone number listed for this group")
+            }
+            
+            else {
```

Comment:
Unless there is a comment before an else statement, I would recommend putting it on the same line as the end bracket like `} else {`